eyieko/landville-frontend
# 167648129 Enable Users to retrieve their deposits

## Description ##
- Retrieve user deposits

## Type of change ##
Please select the relevant option

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##


## How to test: ##
- Clone the repo
- Install dependencies using `npm install`
- Checkout branch ft-retrieve-transactions-167648129
- Navigate to the deposits page using the link `https://landville-frontend-pr-34.herokuapp.com/user/deposits`
- You will see a table of available deposits for a user

## Related Pivotal Tracker stories ##
[#167648129](https://www.pivotaltracker.com/story/show/167648129)
